fendermint/app/src/cmd/genesis.rs

```diff
@@ -213,7 +216,7 @@ fn set_eam_permissions(
 
 fn into_tendermint(genesis_file: &PathBuf, args: &GenesisIntoTendermintArgs) -> anyhow::Result<()> {
     let genesis = read_genesis(genesis_file)?;
-    let genesis_json = serde_json::to_value(&genesis)?;
+    let app_state = hex::encode(std::fs::read(&args.sealed)?);
```

Comment:
Given the sealed state is extremely large - we might benefit from using `base64` instead of hex as that might be more compact. WDYT?

Reply:
maybe a compression before also works. Updated to base64.

Reply:
Because the bytes were all using `hex` through out the codebase, to be consistent. Maybe a compression before encoding will make it smaller?